### ISSUE TITLE
Update for improved accessibility

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/genesis-cc0512#2]
When I ran WAVE against the home page I got 21 Contrast errors in the footer widgets.
- `#999999` on `#333333` does not satisfy WCAG AA or AAA for Normal text size.
- Contrast ratio 4.43:1
- `#9c9c9c` satisfies WCAG AA ( 4.6:1 )
- `#c2c2c2` satisfies WCGA AAA ( 7.09:1 )

There's also a Missing form label on the Search box.

**Comment by bobbingwide** [comment]
I changed the theme in s.b/wordpress and wasn't particularly surprised to receive more Contrast errors.
- The red colour used for links wasn't dark enough.  `#e5554e`
- I changed it to `#a92019`

**Comment by bobbingwide** [comment]
> There's also a Missing form label on the Search box.

- For this I found a blog post entitled Enable Accessibility Features
- https://my.studiopress.com/documentation/snippets/accessibility/enable-accessibility-features/
- I added the code shown to `functions.php`
- Of the CSS examples I copied the CSS used for screen reader text
- I didn't copy Accessible Menu CSS nor the Skip Links CSS
